src/iuse.cpp

```diff
@@ -2089,7 +2089,7 @@ int iuse::marloss(player *p, item *it, bool t, point pos)
                         pgettext("memorial_female", "Opened the Marloss Gateway."));
         p->add_msg_if_player(m_good, _("You wake up in a marloss bush.  Almost *cradled* in it, actually, as though it grew there for you."));
         //~ Beginning to hear the Mycus while conscious: that's it speaking
-        p->add_msg_if_player(m_good, _("unity. together we have reached the door. we provide the final key. now to pass through..."));
+        p->add_msg_if_player(m_good, _("Unity.  Together we have reached the door.  We provide the final key.  Now to pass through..."));
```

Comment:
Much like the slimespawns and the shared-brain communication, the Mycus is a small voice here.  Once you go full Mycus, sure, it's capitalized.